src/utilities.client.js

```diff
@@ -154,7 +154,9 @@ export const markedSetNofollowLinks = (href, title, text) => {
         // Probably invalid url, go on
     }
     const isApifyLink = (urlParsed && /\.apify\.com$/i.test(urlParsed.hostname));
```

Comment:
Shouldn't this be `/(\.|^)apify\.com$/`, to cover also links like https://apify.com/xxx